src/buildercore/trop.py

```diff
@@ -282,7 +282,7 @@ def render_rds(context, template):
         'BackupRetentionPeriod': lu('rds.backup-retention'),
         'DeletionPolicy': lu('rds.deletion-policy'),
         "Tags": tags,
-        "AllowMajorVersionUpgrade": False, # default? not specified.
+        "AllowMajorVersionUpgrade": lu('rds.allow-major-version-upgrade', default=False),
```

Comment:
todo: problem here, I shouldn't have to be using `default=False`, there should already be a default of `False` in the project file. So why isn't `rds.allow-major-version-upgrade` available?

Reply:
Default is missing in the fixtures projects file, I'll add it.

Reply:
sure, I went a bit bonkers over this yesterday and just tidied up what I could this morning and then left it for Friday